tests/integration/pilot/tunneling/external-app/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: external-app
+  template:
+    metadata:
+      labels:
+        app: external-app
+    spec:
+      containers:
+      - name: external-app
+        image: nginx:1.21-alpine
```

Comment:
Do we need nginx? We should use the echo app. If there are missing features we can add them.

Reply:
Done. Now tests use echo apps and nginx is no longer used.